vendor/github.com/hashicorp/go-azure-sdk/resource-manager/applicationinsights/2022-04-01/applicationinsights/README.md

```diff
@@ -77,7 +77,7 @@ if model := read.Model; model != nil {
 
 ```go
 ctx := context.TODO()
-id := applicationinsights.NewResourceGroupID()
+id := applicationinsights.NewResourceGroupID("12345678-1234-9876-4563-123456789012", "example-resource-group")
```

Comment:
we are editing a vendor file?

Reply:
That was not me, that was `go mod tidy` there was an update in main that I had to pull into this branch.